scalding-core/src/main/scala/com/twitter/scalding/typed/memory_backend/MemoryWriter.scala

```diff
@@ -110,7 +111,8 @@ class MemoryWriter(mem: MemoryMode) extends Writer {
     }
     val (id, acts) = idActs
     // now we run the actions:
-    Future.traverse(acts) { fn => fn() }.map(_ => (id, ExecutionCounters.empty))
+    val fut = Future.traverse(acts) { fn => fn() }.map(_ => (id, ExecutionCounters.empty))
+    CFuture(fut, CancellationHandler.empty)
```

Comment:
can we comment how returning an empty cancelation handler from all the executions is working? To me, it seems like `type Action = () => CFuture[Unit]` and we should be composing cancelation. I'm confused how this can work.

Reply:
it's still the case that `type Action = () => Future[Unit]` -- i just wrapped the result in a `CFuture` at the end